src/resources/Catalogs/CatalogInterfaces.ts

```diff
@@ -435,7 +435,7 @@ export interface CatalogObjectType {
     /**
      * If catalog content exist in source
      */
-    hasCatalogContent: boolean;
+    hasIndexedCatalogContent: boolean;
```

Comment:
Shouldn't JSDoc be modified now that we're talking about indexed?

Reply:
"If catalog source has indexed content" ?